src/SFML/Window/WindowImpl.cpp

```diff
@@ -109,6 +109,16 @@ void WindowImpl::setJoystickThreshold(float threshold)
 
 
 ////////////////////////////////////////////////////////////
+bool WindowImpl::setFileDroppingEnabled(bool enabled)
+{
+    // This is the fallback for unsupported platforms.
+    // Trying to enable file dropping returns a bad value
+    // indicating an error, whereas disabling it is fine.
+    return !enabled;
```

Comment:
In my opinion, it doesn't *fail silently*. Disabling it is indistinguishable from it not being present, so why generate an error?